.github/workflows/publish.yml

```diff
@@ -27,23 +30,40 @@ jobs:
         with:
           tag_name: ${{ steps.extract_branch.outputs.branch }}
           release_name: ${{ steps.extract_branch.outputs.branch }}
-      
-  wheel:
+
+  build_wheels:
     name: Build Wheel
     runs-on: ${{ matrix.os }}
-    needs: release
-    
+    needs: setup_release
+
     strategy:
       fail-fast: false
       matrix:
-          # os: [ubuntu-20.04]
-          os: [ubuntu-18.04]
+          os: [ubuntu-20.04, ubuntu-22.04]
           python-version: ['3.7', '3.8', '3.9', '3.10']
-          torch-version: [1.11.0, 1.12.0, 1.12.1]
-          cuda-version: ['113', '116']
+          torch-version: ['1.11.0', '1.12.0', '1.13.0']
+          cuda-version: ['113', '116', '117']
           exclude:
-            - torch-version: 1.11.0
+            # Nvidia only supports 11.7+ for ubuntu-22.04
```

Comment:
These explicit exclude cases seem to be the best method that github actions supports today to deal with these cuda-torch-os compatibility issues. However I could see this getting pretty messy in the future as the dependencies grow in complexity. A better approach would be some boolean relationships to define the matrix - but this would have to be implemented as some sort of external yml generator and adds to the complexity here a bit too much for an initial release.

Leaving it for future work.